pkg/minikube/bootstrapper/bsutil/binaries.go

```diff
@@ -37,6 +37,21 @@ import (
 	"k8s.io/minikube/pkg/minikube/vmpath"
 )
 
+// TransferCrictl transfers the crictl binary to the machine
+func TransferCrictl(cfg config.KubernetesConfig, c command.Runner) error {
+
+	src, err := download.DownloadCrictlBinary(cfg.KubernetesVersion)
+	if err != nil {
+		return errors.Wrapf(err, "downloading crictl")
+	}
+	dir := binRoot(cfg.KubernetesVersion)
+	dst := path.Join(dir, "crictl")
```

Comment:
This path `/var/lib/minikube/binaries/<version>/crictl` is not included in the `PATH` of the machine, so running `crictl` won't find this binary, plus, even if it was you'd be in conflict on which location shows up first in `PATH` the originally installed location or this new location, I'd recommend you overwrite the existing crictl binary (`/usr/bin/crictl`) to ensure it's picked up.